indico/web/templates/forms/checkbox_group_widget.html

```diff
@@ -6,9 +6,15 @@
     {% set field_disabled = input_args.pop('disabled', false) %}
     {% for subfield in field %}
         {% set disabled = field_disabled or subfield.data in field.disabled_choices|default([]) %}
-        <div class="i-checkbox {{ 'checkbox-disabled' if disabled}}">
-            {{ subfield(disabled=disabled, **input_args) }}
-            {{ subfield.label }}
+        {% set reason = field.disabled_choices_reasons.get(subfield.data) %}
+        <div class="i-checkbox">
+            <span class="{{ 'checkbox-disabled' if disabled }}">
+                {{ subfield(disabled=disabled, **input_args) }}
+                {{ subfield.label }}
+            </span>
+            {% if disabled %}
+                <span class="icon-info icon-padding" title="{{ reason }}"></span>
```

Comment:
We use `<i>` for icons everywhere else. Don't forget to update the CSS selector you added in `_inputs.css` ;)

```suggestion
                <i class="icon-info icon-padding" title="{{ reason }}"></i>
```

Reply:
I think `icon-warning` would look better there